src/hypothesis/_settings.py

```diff
@@ -74,12 +74,12 @@ def __delete__(self, obj):
 
     @property
     def __doc__(self):
+        description = all_settings[self.name].description
+        deprecation_message = all_settings[self.name].deprecation_message
         default = repr(getattr(settings.default, self.name)) if \
             self.show_default else '(dynamically calculated)'
-        return '\n'.join((
-            all_settings[self.name].description,
-            'default value: %s' % (default,)
-        ))
+        return '\n\n'.join([description, 'default value: %s' % (default,),
```

Comment:
I'm not massively keen on this change. I don't think that a good warning is necessarily the same as the text we want to include verbatim in the docstring.

Reply:
On the other hand, we demonstrably have trouble keeping documented and programmatic deprecations in sync otherwise.

My preference would be to go back and revise the messages until they work for both!